Jenkinsfile

```diff
@@ -41,4 +41,4 @@
 //@Library(value="pipeline-lib@your_branch") _
 
 // centos8: No matching package to install: 'fio-devel'
-packageBuildingPipelineDAOS(['distros': ['centos7', 'leap15']])
\ No newline at end of file
+packageBuildingPipeline(['distros': ['centos7', 'leap15']])
```

Comment:
Can you remind me why we needed to change this?

Reply:
changed this back, according to you this should work once https://github.com/daos-stack/dpdk/pull/9 is landed

Reply:
Ah, yes, I recall now.  The issue was dpdk, not daos.